qtfred/src/mission/Editor.cpp

```diff
@@ -3183,6 +3183,19 @@ void Editor::normalizeShieldSysData() {
 	}
 }
 
+void Editor::strip_quotation_marks(SCP_string& str) {
+	SCP_string::size_type idx = 0;
+	while ((idx = str.find('\"', idx)) != SCP_string::npos) {
+		str.erase(idx, 1);
+	}
+}
+
+void Editor::pad_with_newline(SCP_string& str, SCP_string::size_type max_size) {
```

Comment:
While `SCP_string::size_type` is obviously correct here it would be simpler to simply use `size_t` to make the code more readable since the size_type will almost always be size_t (and if it isn't there are going to be a lot of other issues).